common/decklist.h

```diff
@@ -13,6 +14,7 @@
 #include <QtCore/QXmlStreamReader>
 #include <QtCore/QXmlStreamWriter>
 #include <common/pb/move_card_to_zone.pb.h>
+#include <utility>
```

Comment:
What are you using from utility?

Reply:
This originates from [bd6098d](https://github.com/Cockatrice/Cockatrice/pull/5171/commits/bd6098d01d059d627af66496aad75f9bde66c60f), which @LunyaticCat authored.

I couldn't find a place it is actually used, along with the newly added QDebug import. I took this opportunity to clean up some unused imports in general in that class.